pkg/datapath/linux/config/config.go

```diff
@@ -595,9 +595,6 @@ is_l3; })`))
 
 func (h *HeaderfileWriter) writeNetdevConfig(w io.Writer, cfg datapath.DeviceConfiguration) {
 	fmt.Fprint(w, cfg.GetOptions().GetFmtList())
-	if option.Config.IsFlannelMasterDeviceSet() {
-		fmt.Fprint(w, "#define HOST_REDIRECT_TO_INGRESS 1\n")
```

Comment:
I think the legacy flannel integration was the only user of `HOST_REDIRECT_TO_INGRESS`, wasn't it (/cc @aanm)? If yes, then we should remove all `#ifdef HOST_REDIRECT_TO_INGRESS` in the datapath.